src/unix/linux_like/linux/mod.rs

```diff
@@ -3173,9 +3455,23 @@ pub const IW_PMKID_LEN: usize = 16;
 
 pub const IW_PMKID_CAND_PREAUTH: ::c_ulong = 0x00000001;
 
+pub const IW_EV_LCP_LEN: usize = ::mem::size_of::<iw_event>() - ::mem::size_of::<iwreq_data>();
+
+pub const IW_EV_CHAR_LEN: usize = IW_EV_LCP_LEN + ::IFNAMSIZ;
+pub const IW_EV_UINT_LEN: usize = IW_EV_LCP_LEN + ::mem::size_of::<u32>();
+pub const IW_EV_FREQ_LEN: usize = IW_EV_LCP_LEN + ::mem::size_of::<iw_freq>();
+pub const IW_EV_PARAM_LEN: usize = IW_EV_LCP_LEN + ::mem::size_of::<iw_param>();
+pub const IW_EV_ADDR_LEN: usize = IW_EV_LCP_LEN + ::mem::size_of::<::sockaddr>();
+pub const IW_EV_QUAL_LEN: usize = IW_EV_LCP_LEN + ::mem::size_of::<iw_quality>();
+
 pub const IW_EV_LCP_PK_LEN: usize = 4;
 
 pub const IW_EV_CHAR_PK_LEN: usize = IW_EV_LCP_PK_LEN + ::IFNAMSIZ;
+pub const IW_EV_UINT_PK_LEN: usize = IW_EV_LCP_PK_LEN + ::mem::size_of::<u32>();
+pub const IW_EV_FREQ_PK_LEN: usize = IW_EV_LCP_PK_LEN + ::mem::size_of::<iw_freq>();
+pub const IW_EV_PARAM_PK_LEN: usize = IW_EV_LCP_PK_LEN + ::mem::size_of::<iw_param>();
+pub const IW_EV_ADDR_PK_LEN: usize = IW_EV_LCP_PK_LEN + ::mem::size_of::<::sockaddr>();
+pub const IW_EV_QUAL_PK_LEN: usize = IW_EV_LCP_PK_LEN + ::mem::size_of::<iw_quality>();
```

Comment:
Same here

Reply:
Can I set a fixed value? Instead of calculating that.

Reply:
If it works, yeah.